---
 GmsCore/Android.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/GmsCore/Android.mk b/GmsCore/Android.mk
index c92839d..8bdc8b3 100644
--- a/GmsCore/Android.mk
+++ b/GmsCore/Android.mk
@@ -32,5 +32,6 @@ LOCAL_OVERRIDES_PACKAGES := com.qualcomm.location
 LOCAL_REQUIRED_MODULES := privapp-permissions-com.google.android.gms.xml default-permissions-com.google.android.gms.xml sysconfig-com.google.android.gms.xml
 LOCAL_PRODUCT_MODULE := true
 LOCAL_USES_LIBRARIES := com.android.location.provider
+LOCAL_OPTIONAL_USES_LIBRARIES := androidx.window.extensions androidx.window.sidecar
 include $(BUILD_PREBUILT)
 

From aa7bb2b364f5d078e743fdcdc7b313c9070e1615 Mon Sep 17 00:00:00 2001
From: danielk43 <11856117+danielk43@users.noreply.github.com>
Date: Sun, 20 Nov 2022 16:28:28 -0500
Subject: [PATCH 2/2] Add manage usb priv-app permission

---
 GmsCore/privapp-permissions-com.google.android.gms.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/GmsCore/privapp-permissions-com.google.android.gms.xml b/GmsCore/privapp-permissions-com.google.android.gms.xml
index 33b9554..aa04b4e 100644
--- a/GmsCore/privapp-permissions-com.google.android.gms.xml
+++ b/GmsCore/privapp-permissions-com.google.android.gms.xml
@@ -7,5 +7,6 @@
         <permission name="android.permission.INSTALL_LOCATION_PROVIDER"/>
         <permission name="android.permission.CHANGE_DEVICE_IDLE_TEMP_WHITELIST"/>
         <permission name="android.permission.UPDATE_APP_OPS_STATS"/>
+        <permission name="android.permission.MANAGE_USB"/>
     </privapp-permissions>
 </permissions>
